packages/babel-parser/src/parser/lval.ts

```diff
@@ -729,6 +740,9 @@ export default abstract class LValParser extends NodeUtils {
 
     const validity = this.isValidLVal(
       type,
+      disallowCallExpression ||
+        (expression.type === "CallExpression" &&
+          expression.callee.type === "Import"),
```

Comment:
We should also exclude `Super` here, could you also a test for the `super()` case?

The `Import` check should be Babel 7 only since we will generate ImportExpression in Babel 8.

Reply:
It suddenly occurred to me that Babel 8 still supports `createImportExpressions` as `false`.